stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ChannelActivity.kt

```diff
@@ -67,6 +67,7 @@ class ChannelActivity : AppCompatActivity() {
          * */
         setContent {
             ChatTheme {
+                ChatTheme.colors.appBackground
```

Comment:
This change looks suspicious

Reply:
That's true, not sure what happened here :D